doc/deploy.md

```diff
@@ -27,6 +27,7 @@ Once your system is ready, collect WAR files in a dedicated directory and rename
     mv ldapadmin-${VERSION}-${PROFILE}.war ldapadmin.war
     mv mapfishapp-${VERSION}-${PROFILE}.war mapfishapp.war
     mv header-${VERSION}-${PROFILE}.war header.war
+    mv geowebcache-webapp-${VERSION}-${PROFILE}.war $WEBAPPS/geowebcache.war
```

Comment:
oups !

WEBAPPS=/var/lib/tomcat6/webapps in my configuration. but it is not useful... i will correct that.